ex03.exs

```diff
@@ -59,8 +59,23 @@ defmodule Ex03 do
 
   """
 
+  #spawning process helper method
+  def spawn_process(chunk, function) do
+   Task.async(Enum, :map, [chunk, function]) 
+  end
+
+  #split helper method
+  def collection_split(collection, number) do
+    length_col = Enum.count(collection)
```

Comment:
No deduction, just a matter of taste, but you could probably combine lines 69 & 70:

~~~ elixir
chunk_size = collection |> Enum.count() |> div(number)
~~~

To my eye it seems slightly more expressive.